### PR TITLE
[FIX JENKINS-46660] Modify creation if no Jenkinsfiles found send to pipeline editor

### PR DESCRIPTION
**Pull request by kzantow**
# Description

Takes the user directly to the pipeilne editor if they don't have jenkinsfiles after going through creation. Covered by updated ATH test.

Just go through creation: git, github, bitbucket, etc. with repositories that don't have jenkinsfiles and you should get dropped right to the editor.

See [JENKINS-46660](https://issues.jenkins-ci.org/browse/JENKINS-46660).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
update to master to get de-flaky stuff

File: blueocean-dashboard/src/main/js/creation/flow2/FlowManager.js [hunk 1]
```diff
@@ -164,6 +164,18 @@ export default class FlowManager {
...
+            setTimeout(() => {
+                this._navigateToPipelineEditor();
+                ToastService.newToast({ text: 'No Jenkinsfiles were found in this repository. Get started by creating one.' });
+            }, this.redirectTimeout);
+        }
+    }
+
+    _navigateToPipelineEditor() {
+        const url = `/organizations/${AppConfig.getOrganizationName()}/pipeline-editor/${encodeURIComponent(this.pipelineName)}/`;
+        this.completeFlow({ url })
```
Comment (cliffmeyers) [inline blueocean-dashboard/src/main/js/creation/flow2/FlowManager.js]: `FlowManager` is meant to be a fairly generic abstraction.. I'm not sure this is the right place for this code.

Comment (kzantow) [reply blueocean-dashboard/src/main/js/creation/flow2/FlowManager.js]: All flows should consistently drop to the editor if it's supported (ideally, that reported status).

Comment (cliffmeyers) [reply blueocean-dashboard/src/main/js/creation/flow2/FlowManager.js]: `FlowManager` is used in Thor's project for flows that have nothing to do with creation.

**Comment by kzantow** [comment]
@cliffmeyers implemented your feedback, PTAL

**Comment by michaelneale** [comment]
OMG WOW WOWOWOW
this was kicking my butt the other week. 

:bee: from me - @cliffmeyers ?

**Pull request merged**